docker/i586-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:17.04
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    cmake \
+    gcc \
+    libc6-dev \
+    make \
+    file \
```

Comment:
Is file dependency required?

Reply:
No, this is a c&p-error from the `i586` file in the `stdsimd` crate.